functions-and-operators/string-functions.md

```diff
@@ -1188,20 +1188,41 @@ Return a string containing octal representation of a number.
 
 ### [`OCTET_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_octet-length)
 
-Synonym for `LENGTH()`.
+Synonym for [`LENGTH()`](#length).
 
 ### [`ORD()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_ord)
 
 Return character code for leftmost character of the argument.
 
 ### [`POSITION()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_position)
 
-Synonym for `LOCATE()`.
+Synonym for [`LOCATE()`](#locate).
 
 ### [`QUOTE()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_quote)
 
 Escape the argument for use in an SQL statement.
 
+If the argument is `NULL`, the function returns `NULL`.
+
+Examples:
+
+For this example you need to use the `--skip-binary-as-hex` option as otherwise it would display the result as hex-encoded instead of directly displaying it.
```

Comment:
```suggestion
To display the result as a hexadecimal-encoded value instead of showing it directly, you need to start the MySQL client with the [`--skip-binary-as-hex`](https://dev.mysql.com/doc/refman/8.0/en/mysql-command-options.html#option_mysql_binary-as-hex) option.
```